app/routes/page.php

```diff
@@ -51,6 +51,7 @@ function renderGuide(Slim $app, Guide $guide, Category $category)
         'guide'              => $guide,
         'linkToEditDocument' => $guide->linkToEdit(),
         'activeMenu'         => $category->getName(),
+        'currentPath'        => $app->environment['PATH_INFO']
```

Comment:
I think this should be $app->request->getPathInfo()

Also global view variables need to be set in initView otherwise it may fail eg on home page, 404 page etc.